lib/v2/stratechery/index.js

```diff
@@ -0,0 +1,20 @@
+const buildData = require('@/utils/common-config');
+
+module.exports = async (ctx) => {
+    const link = 'https://stratechery.com/';
+
+    ctx.state.data = await buildData({
+        link,
+        url: link,
+        title: 'Stratechery by Ben Thompson',
+        author: 'Ben Thompson',
+        description: 'Stratechery provides analysis of the strategy and business side of technology and media, and the impact of technology on society. ',
+        item: {
+            item: 'article',
+            title: `$('article > header > h1 > a').text()`,
+            link: `$('article > header > h1 > a').attr('href')`,
+            pubDate: `Date.parse($('article .entry-date.updated').attr('datetime'))`,
```

Comment:
Some articles do not have `updated` class name
```suggestion
            pubDate: `parseDate($('article .entry-date').attr('datetime'))`,
```